modules/knative-eventing-modifying-consumer-group-IDs-and-topic-names.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * serverless/eventing/tuning/overriding-config-eventing.adoc
+
+:_content-type: PROCEDURE
+[id="knative-eventing-modifying-consumer-group-IDs-and-topic-names_{context}"]
+= Modifying consumer group IDs and topic names
+
+You can change templates for generating consumer group IDs and topic names used by your triggers, brokers, and channels.
+
+.Prerequisites
+
+* You have cluster or dedicated administrator permissions on {ocp-product-title}.
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource (CR) are installed on your {ocp-product-title} cluster.
+* You have created a project or have access to a project that has the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. To change templates for generating consumer group IDs and topic names used by your triggers, brokers, and channels, modify the `KnativeKafka` resource:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: KnativeKafka
+metadata:
+  name: <config-map-name> <1>
+  namespace: knative-eventing
+...
+spec:
+  config:
+    config-kafka-features:
+      triggers.consumergroup.template: <template> <2>
+      brokers.topic.template: <template> <3>
+      channels.topic.template: <template> <4>
+----
+<1> The config map name.
+<2> The template for generating the consumer group ID used by your triggers. Use a valid Go `text/template` value. Defaults to `{% raw %}"knative-trigger-{{ .Namespace }}-{{ .Name }}"{% endraw %}`.
+<3> The template for generating Kafka topic names used by your brokers. Use a valid Go `text/template` value. Defaults to `{% raw %}"knative-broker-{{ .Namespace }}-{{ .Name }}"{% endraw %}`.
+<4> The template for generating Kafka topic names used by your channels. Use a valid Go `text/template` value. Defaults to `{% raw %}"messaging-kafka.{{ .Namespace }}.{{ .Name }}"{% endraw %}`.
++
+.Example template configuration
+[source,yaml]
+----
+apiVersion: v1
+kind: KnativeKafka
+metadata:
+  name: kafka-broker-config
+  namespace: knative-eventing
+...
+spec:
+  config:
+    config-kafka-features:
+      triggers.consumergroup.template: "{% raw %}"knative-trigger-{{ .Namespace }}-{{ .Name }}-{{ .MyCustomValue}}"{% endraw %}"
+      brokers.topic.template: "{% raw %}"knative-broker-{{ .Namespace }}-{{ .Name }}-{{ .MyCustomValue}}"{% endraw %}"
+      channels.topic.template: "{% raw %}"messaging-kafka.{{ .Namespace }}.{{ .Name }}-{{ .MyCustomValue}}"{% endraw %}"
+----
+
+. Apply the config map:
++
+[source,yaml]
+----
+$ oc apply -f <config_map_filename>
```

Comment:
Not a config_map_filename. Probalby better to mention `<knative_kafka_filename>`